crates/evm/src/executor/inspector/access_list.rs

```diff
@@ -68,7 +69,7 @@ impl<DB: Database> Inspector<DB> for AccessListTracer {
             opcode::BALANCE |
             opcode::SELFDESTRUCT => {
                 if let Ok(slot) = interpreter.stack().peek(0) {
-                    let addr: Address = Address::from_slice(&slot.to_be_bytes::<32>()[12..]);
+                    let addr: Address = Address::from_word(slot.to_be_bytes::<32>().into());
```

Comment:
Can also drop `to_be_bytes` because of [`U256: From<B256>`](https://docs.rs/alloy-primitives/latest/alloy_primitives/aliases/type.B256.html#impl-From%3CUint%3C256,+4%3E%3E-for-B256)